portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClaySmoke.testcase

```diff
@@ -0,0 +1,57 @@
+@component-name = "portal-frontend-infrastructure"
+definition {
+
+	property osgi.modules.includes = "frontend-taglib-clay-sample-web";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.component.names = "Clay";
+	property testray.main.component.name = "User Interface";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		JSONLayout.addPublicLayout(
+			groupName = "Guest",
+			layoutName = "Clay Sample Test Page");
+
+		JSONLayout.updateLayoutTemplateOfPublicLayout(
+			groupName = "Guest",
+			layoutName = "Clay Sample Test Page",
+			layoutTemplate = "1 Column");
+
+		JSONLayout.addWidgetToPublicLayout(
+			groupName = "Guest",
+			layoutName = "Clay Sample Test Page",
+			widgetName = "Clay Sample");
+
+		Navigator.gotoPage(pageName = "Clay Sample Test Page");
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			JSONLayout.deletePublicLayout(
+				groupName = "Guest",
+				layoutName = "Clay Sample Test Page");
+		}
+	}
+
+	@description = "Check if each nav section of clay sample portlet looks normal from screenshot"
+	@priority = "3"
+	test CanRenderUI {
+		task ("Change tabs and take a screenshot") {
+			for (var tabs : list "Alerts,Badges,Buttons,Cards,Dropdowns,Form Elements,Icons,Labels,Links,Management Toolbars,Navigation Bars,Pagination Bars,Progress Bars,Stickers") {
+				Navigator.gotoNavTab(navTab = "${tabs}");
+
+				takeScreenshot();
+			}
```

Comment:
Unfortunately, it doesn't look like this for loop is going to translate well for our poshi report.

![image](https://user-images.githubusercontent.com/35235266/153522837-f229f96a-3c27-4380-967f-7bc38bd423b9.png)

The poshi report seems to only save the last screenshot from the loop.

Maybe you should try making a new macro that navigates and takes screenshot, then have the for loop call that macro. Check that your local poshi report shows all screenshots in each tab.